workflows/notes.sh

```diff
@@ -0,0 +1,520 @@
+#     obj = cls.from_string(defstr)
+#   File "/opt/conda/lib/python3.8/site-packages/xclim/sdba/base.py", line 82, in from_string
+#     params[dd["name"]] = cls._parametrizable_classes[
+# KeyError: 'Grouper'
+
+
+
+argo submit dc6-dev-workflow.yaml --entrypoint rechunk \
+    -p variable="fakevariable" \
+    -p in-zarr="clean-dev/ACCESS-ESM1-5.historical.zarr" \
+    -p out-zarr="scratch/rechunk-dev/output.zarr" \
+    -p time-chunk="-1" \
+    -p lat-chunk="30" \
+    -p lon-chunk="30" \
+    -n argo --watch
+# dc6-dev-plrg4-580946482: Traceback (most recent call last):
+# dc6-dev-plrg4-580946482:   File "/argo/staging/script", line 25, in <module>
+# dc6-dev-plrg4-580946482:     plan = rechunk(
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/rechunker/api.py", line 296, in rechunk
+# dc6-dev-plrg4-580946482:     copy_spec, intermediate, target = _setup_rechunk(
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/rechunker/api.py", line 377, in _setup_rechunk
+# dc6-dev-plrg4-580946482:     copy_spec = _setup_array_rechunk(
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/rechunker/api.py", line 497, in _setup_array_rechunk
+# dc6-dev-plrg4-580946482:     target_array = _zarr_empty(
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/rechunker/api.py", line 151, in _zarr_empty
+# dc6-dev-plrg4-580946482:     return store_or_group.empty(
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/hierarchy.py", line 901, in empty
+# dc6-dev-plrg4-580946482:     return self._write_op(self._empty_nosync, name, **kwargs)
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/hierarchy.py", line 661, in _write_op
+# dc6-dev-plrg4-580946482:     return f(*args, **kwargs)
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/hierarchy.py", line 907, in _empty_nosync
+# dc6-dev-plrg4-580946482:     return empty(store=self._store, path=path, chunk_store=self._chunk_store,
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/creation.py", line 227, in empty
+# dc6-dev-plrg4-580946482:     return create(shape=shape, fill_value=None, **kwargs)
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/creation.py", line 121, in create
+# dc6-dev-plrg4-580946482:     init_array(store, shape=shape, chunks=chunks, dtype=dtype, compressor=compressor,
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/storage.py", line 348, in init_array
+# dc6-dev-plrg4-580946482:     _init_array_metadata(store, shape=shape, chunks=chunks, dtype=dtype,
+# dc6-dev-plrg4-580946482:   File "/opt/conda/lib/python3.8/site-packages/zarr/storage.py", line 377, in _init_array_metadata
+# dc6-dev-plrg4-580946482:     raise ContainsArrayError(path)
+# dc6-dev-plrg4-580946482: zarr.errors.ContainsArrayError: path 'lat' contains an array
```

Comment:
should we remove this too?

Reply:
Yeah, thanks. I'm removing this entire file.

Marking resolved.